### PR TITLE
upgrade minikube to 0.35.0

### PR DESCRIPTION
**Pull request by yudong2015**
upgrade:
minikube == 0.35.0
kubernetes == 1.13.4
kubectl == 1.13.4

### CODE REVIEW

File: deploy/kubernetes/scripts/init-helm.sh [hunk 1]
```diff
@@ -3,13 +3,14 @@
 TILLER_DEPLOY=`kubectl get deploy -n kube-system | grep 'tiller-deploy'`
 [ -n "$TILLER_DEPLOY" ] && echo "Helm is already initialized." && exit 1
 
+export DESIRED_VERSION=v2.12.0
 
-curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | DESIRED_VERSION=v2.12.0 bash
+curl https://raw.githubusercontent.com/helm/helm/${DESIRED_VERSION}/scripts/get | bash
 
-helm init --tiller-image=gcr.io/kubernetes-helm/tiller:v2.12.0 --upgrade
+helm init --tiller-image=gcr.io/kubernetes-helm/tiller:${DESIRED_VERSION}
```
Comment (chilianyi) [inline deploy/kubernetes/scripts/init-helm.sh]: If tiller already exist, need --upgrade

File: deploy/kubernetes/scripts/install-minikube.sh [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/${MINIKUBE_VERSION}/minikube-linux-amd64 \
   && chmod +x minikube && sudo mv minikube /usr/local/bin/
 
-sudo minikube start --vm-driver=none --kubernetes-version=v1.10.0 --extra-config=apiserver.Authorization.Mode=RBAC
+sudo minikube start --vm-driver=none --kubernetes-version=${KUBE_VERSION} --extra-config=apiserver.v=10 --extra-config=kubelet.max-pods=100
 
 sudo minikube update-context
 
-sudo kubectl create clusterrolebinding add-on-cluster-admin --clusterrole=cluster-admin --serviceaccount=kube-system:default
+sudo chown -R ${USER}:${USER} ${HOME}/.kube
```
Comment (chilianyi) [inline deploy/kubernetes/scripts/install-minikube.sh]: Where is the ${HOME} defined?

File: .travis.yml [hunk 3]
```diff
@@ -19,15 +20,17 @@ addons:
   apt:
     packages:
     - docker-ce
+    - socat
+    - ebtables
```
Comment (chilianyi) [inline .travis.yml]: What is ebtables used for?

Comment (yudong2015) [reply .travis.yml]: Minikube v0.35.0 requires ebtables.

File: deploy/kubernetes/scripts/install-minikube.sh (hunk 2 above)
Comment (yudong2015) [reply deploy/kubernetes/scripts/install-minikube.sh]: ${HOME} is system env, means the home dir of current user.

Comment (chilianyi) [reply deploy/kubernetes/scripts/install-minikube.sh]: OK

File: .travis.yml (hunk 3 above)
Comment (chilianyi) [reply .travis.yml]: OK

File: deploy/kubernetes/scripts/init-helm.sh (hunk 1 above)
Comment (yudong2015) [reply deploy/kubernetes/scripts/init-helm.sh]: This script is used for travis, so the tiller not exist; But there is no harm if add it, I would add it.

**Pull request merged**